english/explore-outside/help-child.md

```diff
@@ -0,0 +1,13 @@
+As you approach the small, green child fishing in the marshmallow boat, you realize
+that, indeed, the marshmallow boat is beginning to melt.
+Since it is melting slowly, you have time to intervene.
+
+Do you:
+
+[return to the marshmallow room to eat more marshmallows because you no longer
+feel the need to help?](../eating-marshmallows/eating-marshmallow.md)
+
+[head now to the tower because you can see the child is safe in this one foot
+deep lake?](./journey-to-flaming-tower/journey-to-flaming-tower.md)
+
+[give the child your extra marshmallow to repair the boat?](./repair-boat/repair-boat.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.